### ISSUE TITLE
Manage Templates/Customer Overview is mission in the documentation

### ISSUE THREAD

**Issue by sundas-dev** [CloudplexPlatform/docs#15]
There is no overview given for Manage Templates section which is in top bar of dev.cloudplex.io. And also customer templates documentation is missing.

**Comment by bakhtawermehmood** [comment]
yes this is missing in documentation but we have removed customer templates option for now. 
default configurations are given for network and cluster. 
![image](https://user-images.githubusercontent.com/17968166/86447682-23cd6f80-bd2f-11ea-9b6d-ad6fe77033ba.png)

**Comment by sundas-dev** [comment]
But Manage Templates option is still there.
